### DISCUSSION TITLE
Semi-abandoned?

### DISCUSSION THREAD

**Discussion by MrSuttonmann** [louislam/dockge#500] [General]
I really love Dockge, but I can’t help notice the lack of activity over the past few months. There’s been one or two PR merges but not much else.

Just wondering if @louislam is still interested in actively maintaining this excellent project?

🐳

**Comment by cyril59310** [comment]
The project is still active, however, the maintainer is primarily focused on his other project, [UptimeKuma](https://github.com/louislam/uptime-kuma), on version 2.0 for the moment, as indicated [here](https://github.com/louislam/dockge/pull/414#discussion_r1480587671).

**Reply by cabinlab** [reply]
Was wondering the same! UptimeKuma 2.0.0 is currently listed as 95% complete, so progress continues

https://github.com/louislam/uptime-kuma/milestones
